python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -951,11 +951,19 @@ def test_union_classmethod_usage(self):
     def test_isinstance_dataframe(self):
         self.assertIsInstance(self.spark.range(1), DataFrame)
 
-    def test_checkpoint_dataframe(self):
+    def test_local_checkpoint_dataframe(self):
```

Comment:
note: there are no tests at all for reliable checkpoint in pyspark API. I renamed this test accordingly.